backend/app/adapter/sqldb/short_link.go

```diff
@@ -286,6 +286,32 @@ func (s ShortLinkSQL) composeParamList(numParams int) string {
 	return parameterStr
 }
 
+// DeleteShortLink deletes an existing user short link.
+func (s ShortLinkSQL) DeleteShortLink(alias string) error {
+	query := fmt.Sprintf(`
+	DELETE FROM %s
+	WHERE %s=$1
+	`,
+		table.ShortLink.TableName,
+		table.ShortLink.ColumnAlias,
+	)
+
+	result, err := s.db.Exec(query, alias)
+	if err != nil {
+		return err
+	}
+
+	affectedRowCount, err := result.RowsAffected()
+	if err != nil {
+		return err
+	}
+	if affectedRowCount == 0 {
+		return repository.ErrShortLinkDeletionFailure{Alias: alias}
```

Comment:
```suggestion
		return repository.ErrEntryNotFound{Alias: alias}
```

Reply:
I have a question: should this actually be an error?

Reply:
I believe so. How will we know if we attempted to delete a short link but it was not found?

Reply:
`ErrEntryNotFound` is a string type so the error returned would look like this.
```Golang
var err repository.ErrEntryNotFound = "error message here"
return err
```
It looks a little weird :sweat_smile:

Reply:
@oatovar How about `ErrAliasNotFound`?

Reply:
Want to make it reusable across repositories while conveying the accurate meaning.

Reply:
That's perfect :smile: just updated it to that.